stmhal/boards/STM32F429DISC/pins.csv

```diff
@@ -0,0 +1,38 @@
+SW,PA0
+PA0,PA0
+PA5,PA5
+PA8,PA8
+PA9,PA9
+PA10,PA10
+PA13,PA13
+PA14,PA14
```

Comment:
I generally include any accessible pin. The only disadvantage of including a pin is that it takes up a bit of flash.

For example, if the SDRAM chip isn't initialized that all of those pins could be used as GPIOs (even though the SDRAM chip is connected, if the chip select line isn't selected then it won't be driving those pins). Similarly with the LCD. It has an ENABLE line which could be driven low and then all of the RGB pins could be used as GPIO.